src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -52,12 +62,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <!--
-      We check in the Release / Production build of blazor.*.js, but not the Debug builds. Consequently the former is always available to embed, the latter is only available
-      if Web.JS was built locally. Use the Debug build when available and building in Debug configuration.
-    -->
-    <BlazorServerJSFile>..\..\Web.JS\dist\Release\blazor.server.js</BlazorServerJSFile>
-    <BlazorServerJSFile Condition="'$(Configuration)' == 'Debug' AND Exists('..\..\Web.JS\dist\Debug\blazor.server.js')">..\..\Web.JS\dist\Debug\blazor.server.js</BlazorServerJSFile>
+    <BlazorServerJSFile>..\..\Web.JS\dist\$(Configuration)\blazor.server.js</BlazorServerJSFile>
```

Comment:
Does this fail if the file isn't present? I **really** want to make sure that this doesn't regress again.

Reply:
Yes it fails. Though I didn't bother cleaning up the error message: 
```
CSC : error CS1566: Error reading resource '_framework/blazor.server.js' -- 'Could not find a part of the path 'C:\gh\AspNetCore\src\Components\Web.JS\dist\Debug\blazor.server.js'.' [C:\gh\AspNetCore\src\Components\Server\src\Microsoft.AspNetCore.Components.Server.csproj]
```

Reply:
Good enough for me!